test/CMakeLists.txt

```diff
@@ -2,5 +2,4 @@
 add_executable(testregex testregex.cpp)
 add_test(testregexcorrect testregex)
 
-
 # correctness testing of checks
```

Comment:
Sounds right. Later you can also define a cmake macro `add_tidy_test` that saves some typing and just takes the 2 argument: filename + test.